### ISSUE TITLE
Way to much REST Prometheus metrics

### ISSUE THREAD

**Issue by piotrkochan** [RocketChat/Rocket.Chat#12608]
### Description:

We are running prometheus metrics in order to monitor some values but recently we've started to receive that service is down. After investigation we noticed that yes, metrics are available but there is so much of them that Prometheus takes too much time to scrape them.

There is a lot of metrics like:

```
rocketchat_rest_api{unique_id="DNFAusNa7YFNZgsQJ",site_url="https://rocket",version="v1",quantile="0.5",method="get",user_agent="RC Mobile; Android 8.1.0; v2.6.1 (2043)",entrypoint="groups.messages",status="200"} 0.004411679
rocketchat_rest_api{unique_id="DNFAusNa7YFNZgsQJ",site_url="https://rocket",version="v1",quantile="0.9",method="get",user_agent="RC Mobile; Android 8.1.0; v2.6.1 (2043)",entrypoint="groups.messages",status="200"} 0.004411679
rocketchat_rest_api{unique_id="DNFAusNa7YFNZgsQJ",site_url="https://rocket",version="v1",quantile="0.95",method="get",user_agent="RC Mobile; Android 8.1.0; v2.6.1 (2043)",entrypoint="groups.messages",status="200"} 0.004411679
rocketchat_rest_api{unique_id="DNFAusNa7YFNZgsQJ",site_url="https://rocket",version="v1",quantile="0.99",method="get",user_agent="RC Mobile; Android 8.1.0; v2.6.1 (2043)",entrypoint="groups.messages",status="200"} 0.004411679
rocketchat_rest_api{unique_id="DNFAusNa7YFNZgsQJ",site_url="https://rocket",version="v1",quantile="0.999",method="get",user_agent="RC Mobile; Android 8.1.0; v2.6.1 (2043)",entrypoint="groups.messages",status="200"} 0.004411679
```

It looks like when someone else, with different device will use our server then a lot of more metrics gonna be generated. Right now /metrics endpoint response is above 5M...

### Expected behavior:

I wish there is a switch to disable REST API metrics.

### Server Setup Information:

- Version of Rocket.Chat Server: 0.69.2
- Operating System: Ubuntu
- Deployment Method: TAR
- Number of Running Instances: 1
- DB Replicaset Oplog: 0
- NodeJS Version: 8.11
- MongoDB Version:

**Comment by reetp** [comment]
This is a NFR. Please open it in the appropriate place
https://forums.rocket.chat/c/feature-requests

**Comment by reetp** [comment]
Also, always test on the latest version to see if your issue has been fixed - 0.69.2 is a two months behind current 0.71.1 and there may be a lot of changes

**Comment by reetp** [comment]
@rocket-cat close
